packages/contracts/contracts/CdpManagerStorage.sol

```diff
@@ -163,17 +163,17 @@ contract CdpManagerStorage is EbtcBase, ReentrancyGuard, ICdpManagerData, AuthNo
     uint256 public stakingRewardSplit;
 
     // The timestamp of the latest fee operation (redemption or new EBTC issuance)
-    uint256 public lastRedemptionTimestamp;
+    uint128 public lastRedemptionTimestamp;
 
-    mapping(bytes32 => Cdp) public Cdps;
+    mapping(bytes32 => CdpStorage) public cdpStorages;
```

Comment:
I think you can pack the storage a bit more if you move cdpStorages up so that it's before lastRedemptionTimestamp. I got 20 vs 19 slots from foundry inspect.